app/models/weet.js

```diff
@@ -17,7 +17,8 @@ module.exports = (sequelize, DataTypes) => {
     },
     {
       tableName: 'weets',
-      underscored: true
+      underscored: true,
+      updatedAt: false
```

Comment:
Why?

Reply:
that's because the updated_at field is not in the weets table, I made this approach because I think our weets should not be updated